Source/Scene/LabelCollection.js

```diff
@@ -103,6 +125,44 @@ define([
         var glyphIndex;
         var textIndex;
 
+        var showBackground = label._showBackground;
+        var backgroundBillboard = label._backgroundBillboard;
+        if (!showBackground) {
```

Comment:
I'm suggesting a fairly trivial change:
```javascript
var spareBackgroundBillboards = labelCollection._spareBackgroundBillboards;
// use spareBackgroundBillboards below
```
Just a style thing from the Coding Guide: https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#basic-code-construction